### PR TITLE
unanchors the boxstation courtroom gavel block

### PR DESCRIPTION
**Pull request by PKPenguin321**

### CODE REVIEW

**Comment by TechnoAlchemisto** [comment]
:controversial:

**Comment by Aloraydrel** [comment]
Killing the meta

File: _maps/map_files/TgStation/tgstation.2.1.3.dmm [hunk 1]
```diff
@@ -5225,7 +5225,7 @@
 "ajR" = (
 /obj/structure/table/wood,
 /obj/item/weapon/gavelblock{
-	anchored = 1
+	anchored = 0
```
Comment (xxalpha) [inline _maps/map_files/TgStation/tgstation.2.1.3.dmm]: PK you goof, just remove the var edit.

**Comment by Zilenan91** [comment]
The metastation one will also require freedom

Anyways it's nice a stealth change is finally being reverted :+1:

File: _maps/map_files/TgStation/tgstation.2.1.3.dmm (hunk 1 above)
Comment (PKPenguin321) [inline _maps/map_files/TgStation/tgstation.2.1.3.dmm]: o

**Comment by PKPenguin321** [comment]
>The metastation one will also require freedom

it has always had freedom

also merge conflicts noooo*oooo*

**Comment by Zilenan91** [comment]
I'm p sure someone shackled the metastation one a while ago.

**Comment by PKPenguin321** [comment]
I checked the map file, it's not anchored already :)

**Pull request merged**